pkg/pf/tests/provider_configure_test.go

```diff
@@ -59,20 +109,60 @@ func TestConfigure(t *testing.T) {
 func TestConfigureInvalidTypes(t *testing.T) {
 	t.Setenv("PULUMI_DEBUG_YAML_DISABLE_TYPE_CHECKING", "true")
 
-	t.Run("bool-type-conversion", crosstests.MakeConfigure(
+	t.Run("string-as-bool", crosstests.MakeConfigure(
```

Comment:
Yet another dimension maybe to worry about, but we recently had a problem here around requesting the type to change at Pulumi level (skipMetadataAPiCheck). Perhaps another PR. https://github.com/pulumi/pulumi-terraform-bridge/issues/2520